docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 services:
   fides:
     image: ethyca/fides:local
-    command: uvicorn --host 0.0.0.0 --port 8080 --reload fides.api.main:app
+    command: uvicorn --host 0.0.0.0 --port 8080 --reload --reload-dir src fides.api.main:app
```

Comment:
@sanders41 — this is the cause of the issue we noticed a few weeks back where the server took much longer to come up (because we were waiting for watchgod to start). Credit to @NevilleS for this!

Reply:
oof, thank you for this!